### PR TITLE
Create autoyast installation for textmode in yast job group

### PR DESCRIPTION
**Pull request by manfredi**
- Description: Create autoyast installation for textmode in _yast_ job group
- Related ticket: https://progress.opensuse.org/issues/130985
- Verification run:
  + [overview](https://openqa.suse.de/tests/overview?distri=sle&build=manfredi%2Fos-autoinst-distri-opensuse%23issues-130985&version=15-SP5)
  + [create_hdd_textmode_autoyast@ppc64le](https://openqa.suse.de/tests/overview?distri=sle&version=15-SP5&build=102.1&groupid=96)
- Related [MR#519](https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/519)

### CODE REVIEW

**Comment by lemon-suse** [comment]
In general, LGTM, just for the VR of ppc64le, it should be on ppc64le-spvm and please update MR for it also.

File: data/yam/autoyast/create_hdd_textmode_aarch64.xml [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+          <create_subvolumes t="boolean">true</create_subvolumes>
+          <filesystem t="symbol">btrfs</filesystem>
+          <format t="boolean">true</format>
+          <mount>/</mount>
+          <mountby t="symbol">uuid</mountby>
+          <partition_id t="integer">131</partition_id>
+          <partition_nr t="integer">2</partition_nr>
+          <quotas t="boolean">true</quotas>
+          <resize t="boolean">false</resize>
+          <size>29928456192</size>
```
Comment (rfan1) [inline data/yam/autoyast/create_hdd_textmode_aarch64.xml:179]: How about set the size to `max`?

Comment (lemon-suse) [reply data/yam/autoyast/create_hdd_textmode_aarch64.xml:179]: Yes, set to max can avoid risk here than hardcode.

Comment (lemon-suse) [reply data/yam/autoyast/create_hdd_textmode_aarch64.xml]: Yes, only for root folder ('/').

File: data/yam/autoyast/create_hdd_textmode_ppc64le.xml [hunk 2]
```diff
@@ -0,0 +1,467 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (lemon-suse) [inline data/yam/autoyast/create_hdd_textmode_ppc64le.xml:1]: It is different for power kvm and power VM, so it is better to update the name, such as:
data/yam/autoyast/create_hdd_textmode_powervm.xml

Comment (rfan1) [reply data/yam/autoyast/create_hdd_textmode_ppc64le.xml:1]: > It is different for power kvm and power VM, so it is better to update the name, such as: data/yam/autoyast/create_hdd_textmode_powervm.xml

As I know, Yast team will not test power + kvm backend any more, then you don't need to change the name.

or if kvm is still tested, I would suggest keeping a single autoyast file for kvm backend/spvm/powerhmc, there should be 2 differences for kvm backend:

1.  set the root disk to `/dev/vda`
2.  we don't need to enable sshd service in firewalld

Comment (lemon-suse) [reply data/yam/autoyast/create_hdd_textmode_ppc64le.xml:1]: For power kvm we still need it for yam migration test. For this ticket, we have a power vm profile is enough for current yast job group test. So I think we can rename it to powervm for now, for power kvm we can work on it on another ticket, and it should be for migration test on SLE12SP5.

**Review by lemon-suse — APPROVED** [review]

**Review by rfan1 — APPROVED** [review]
LGTM

**Pull request merged**